cmd/playground/Dockerfile

```diff
@@ -20,8 +20,10 @@ ARG NUCLIO_LABEL
 
 FROM nuclio-base:$NUCLIO_LABEL as build-binary
 
+ARG NUCLIO_GO_LINK_FLAGS_INJECT_VERSION
+
 # build the playground
-RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -a -installsuffix cgo -ldflags="-s -w" -o playground cmd/playground/main.go
+RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -a -installsuffix cgo -ldflags="${NUCLIO_GO_LINK_FLAGS_INJECT_VERSION}" -o playground cmd/playground/main.go
```

Comment:
multiline this like you did in the other dockerfiles

Reply:
No need, I am about remove playground soon
Playground is obsolete (it was used as nuclio early-days dashboard)

Reply:
deleted > https://github.com/nuclio/nuclio/pull/1706